examples/using-router/components/CustomLink.js

```diff
@@ -0,0 +1,28 @@
+import { withRouter } from 'next/router'
+
+function onClickHandler (href, router) {
+  return (e) => {
+    e.preventDefault()
+    router.push(href)
+  }
+}
+
+const CustomLink = ({ children, href, router }) => (
+  <a
+    className={router.pathname === href ? 'active' : ''}
+    href='#'
```

Comment:
Here you can put href

```
href={href}
```

The `event.preventDefault()` will prevent the link from working, but it is good for the user to see the link when hovering.